UN-OCHA/ocha_basic
# CD-65: Use genuine Drupal output for User Menu components

## [CD-65: Use genuine Drupal output for User Menu components](https://humanitarian.atlassian.net/browse/CD-65)

We faced a lot of difficulty reconciling the non-Drupal styleguide with actual implementations. This PR brings our Global Header much more in line with the site's we've already converted to Common Design.

## Testing

- If it doesn't exist, add a "Login" link to User Menu, its path should be `user/login`
- If it doesn't exist, add a "Logout" link to User Menu, its path should be `user/logout`
- Place **User Menu** Block inside the **Header Top** Region

Hopefully something like this comes out:

### Logged out, anonymous visitor
<img width="1401" alt="cd-global-header-user-menu-anon" src="https://user-images.githubusercontent.com/254753/44398518-670c0400-a544-11e8-9c20-72d61aa195ac.png">

### Logged in, authenticated user
<img width="1400" alt="cd-global-header-user-menu" src="https://user-images.githubusercontent.com/254753/44398490-45ab1800-a544-11e8-8e64-57bfb2f00590.png">

## Testing language dropdown

- Enable **Locale** module
- Add a language to your installation
- Set **URL** as your language selection method

The Language dropdown should now appear without any other theming config.

<img width="1061" alt="cd-global-header-language-switcher" src="https://user-images.githubusercontent.com/254753/44398603-c9650480-a544-11e8-99c2-4c35101d045e.png">
